manifests/s/Shure/WirelessWorkbench/6.15.0/Shure.WirelessWorkbench.locale.en-US.yaml

```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Shure.WirelessWorkbench
+PackageVersion: 6.15.0
+PackageLocale: en-US
+Publisher: Shure
+PublisherUrl: https://www.shure.com/en-US/products/software/wwb6
+PublisherSupportUrl: https://pubs.shure.com/quickstart/Wireless-Workbench/en-US
+PrivacyUrl: https://www.shure.com/en-US/meta/privacy
+Author: Shure
+PackageName: Wireless Workbench
```

Comment:
```suggestion
PackageName: Wireless Workbench 6
```

Reply:
It would probably be fine without the `6` but this is for stronger matching for `winget list`.